content/en/docs/tasks/deployment/voms_importer/_index.md

```diff
@@ -63,10 +52,13 @@ or role in VOMS; those are not considered by the importer script.
 
 ## Migration of users
 
-Only **active** VOMS users are migrated. **A user suspended in VOMS, may not be
-also suspended in IAM.** Since IAM at CERN is also integrated with the CERN
+Only **active** VOMS users are imported: nothing is done on the INDIGO IAM side
```

Comment:
I see that there is a possibility to import (and also synchronise) suspended users from VOMS to IAM via a command line flag. If you pass `-synchronise-activation-status` when running the script, the suspended user from VOMS is imported and the users' status and end time are also synchronised ->
https://github.com/indigo-iam/voms-importer/blob/main/vomsimporter.py#L1099

Reply:
@rmiccoli thanks for your comment. In fact I didn't check but just rephrased what was said before. I'm happy to document `--synchronize-activatation-status`!

Reply:
Hopefully clarified!